.tekton/cli-build.yaml

```diff
@@ -0,0 +1,391 @@
+apiVersion: tekton.dev/v1
```

Comment:
I like this but would have been nice to review it in its own PR.

Reply:
Yeah, I agree, I wanted to get this done...